meta-resin-morty/classes/image_types_resin.bbclass

```diff
@@ -0,0 +1,297 @@
+inherit image_types
```

Comment:
We could look and see if the image we need is in either of these directories. But it seems like a hack to me. Don't know of other way as of now.
Any suggestions?